packages/css/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@markprompt/css",
+  "version": "0.0.0",
+  "description": "Common CSS for Markprompt components",
+  "license": "MIT",
+  "type": "module",
+  "exports": "./markprompt.css",
```

Comment:
Since this package only contains CSS, maybe it’s best to loosen up a bit and not specify module type / exports.

```suggestion
```